pandas/tests/frame/test_arithmetic.py

```diff
@@ -1536,3 +1536,14 @@ def test_dataframe_blockwise_slicelike():
 
         expected = pd.DataFrame({i: left[i] + right[i] for i in left.columns})
         tm.assert_frame_equal(res, expected)
+
+
+def test_dataframe_operation_with_non_numeric_types():
+    # GH #22663
+    df = pd.DataFrame([[1, 2], [4, 5]], columns=list("ab"))
+    result = df + pd.Series([-1], index=list("a"))
+    expected = pd.DataFrame([[0.0, np.nan], [3.0, np.nan]], columns=list("ab"))
+    tm.assert_frame_equal(result, expected)
+    df["b"] = "b"
```

Comment:
add a blank line here

Reply:
I have rewrite the tests with a fixture. It is then clearer about `result` and `expected`.